data-syncs/supported-data-sync-sources/rest-api.md

```diff
@@ -81,6 +93,34 @@ You can learn more about these sections in [Appendix A - Other Sections.](rest-a
 
 <figure><img src="../../.gitbook/assets/image (298).png" alt=""><figcaption><p>Image 2: The Source Tab</p></figcaption></figure>
 
+## Best practices
+
+For JSON objects, use `$.data` in the **Records Root JSONPAth** to access object values. Otherwise, you must append each column in the **Schema** section with `$.data`. 
```

Comment:
Same thing here; remove. This is only true if the response object looks like

```json
{
  "data": { ... }
}
```

Reply:
I set up some mock data to try to get this .data stuff but I can't seem to make it work